contracts/deploy/mainnet/138_strategist_does_buyback.js

```diff
@@ -0,0 +1,71 @@
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "138_strategist_does_buyback",
+    forceDeploy: false,
+    // forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    // proposalId: "",
+  },
+  async ({ deployWithConfirmation }) => {
+    // Proxy contracts
+    const cOUSDBuybackProxy = await ethers.getContract("BuybackProxy");
+    const cOETHBuybackProxy = await ethers.getContract("OETHBuybackProxy");
+    const cARMBuybackProxy = await ethers.getContract("ARMBuybackProxy");
+
+    // Old buyback contracts
+    const cOldOUSDBuyback = await ethers.getContractAt(
+      ["function updateBuybackSplits() external"],
+      cOUSDBuybackProxy.address
+    );
+    const cOldOETHBuyback = await ethers.getContractAt(
+      ["function updateBuybackSplits() external"],
```

Comment:
And this should probably be `OETHBuyback`

Reply:
This has ABI for the same reasons as above: https://github.com/OriginProtocol/origin-dollar/pull/2506#discussion_r2075177993